lib/libzutil/os/windows/zutil_import_os.c

```diff
@@ -621,10 +623,13 @@ zpool_find_import_blkid(libpc_handle_t *hdl, pthread_mutex_t *lock,
 
 				slice = zutil_alloc(hdl, sizeof (rdsk_node_t));
 
+				uint64_t size = GetFileDriveSize(disk);
+
 				error = asprintf(&slice->rn_name,
 				    "#%llu#%llu#%s",
-				    0ULL, GetFileDriveSize(disk),
+				    0ULL, size,
```

Comment:
I appear to be getting some funny business where this gets truncated to INT_MAX (so I get paths starting with `#0#2147483648`) unless you are more explicit.

Reply:
could be missing prototype for GetFileDriveSize(), defaults to int